stock_reception_screen_mrp_subcontracting/models/stock_reception_screen.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import models
+
+
+class StockReceptionScreen(models.Model):
+    _inherit = "stock.reception.screen"
+
+    def _split_move(self, move, qty):
+        # Overridden to remove the link between the move of finished product
+        # and the move to receive to ease the split
+        move_is_subcontract = move.is_subcontract
```

Comment:
shouldn't the rest run only if is subcontract as done for the other methods?

Reply:
We could do that yes to make things clearer (it doesn't change anything because we're only updating the cached value with the original one here).